python/amici/petab_objective.py

```diff
@@ -0,0 +1,236 @@
+"""Functionality related to evaluating the objective function for a PEtab
+problem"""
+
+import numbers
+import numpy as np
+import pandas as pd
+import copy
+
+from typing import List, Sequence
+import amici
+import petab
+
+
+def edatas_from_petab(
+        model: amici.Model, measurement_df: pd.DataFrame,
+        condition_df: pd.DataFrame,
+        simulation_conditions=None) -> List[amici.ExpData]:
+    """
+    Create list of amici.ExpData objects for PEtab problem.
+
+    Arguments:
+        model:
+            AMICI model
```

Comment:
sometimes a full stop, sometimes not 😬

Reply:
Generally putting them, but not too fanatic about it.